web/modules/custom/farm_nfa/js/farmOS.map.behaviors.farm_nfa_plan_locations_layers.js

```diff
@@ -1,12 +1,73 @@
 (function () {
   farmOS.map.behaviors.farm_nfa_plan_locations_layers = {
-    attach: function (instance) {
+    attach: async function (instance) {
       var url = new URL('/nfa-assets/geojson/' + instance.farmMapSettings.plan, window.location.origin + drupalSettings.path.baseUrl)
       var newLayer = instance.addLayer('geojson', {
         title: Drupal.t('Locations'),
         url,
         color: 'orange',
       })
+
+      // Add a layer for fire alerts in the GFW plan tab
+      let gfwRegex = a = /\/plan\/\d+\/gfw/;
```

Comment:
Not sure if I understand what this does?

Reply:
@pcambra this is a typo, this must have come while refactoring things out, have reverted this change, apologies will be more careful in the future